### ISSUE TITLE
Auto download missing fixer

### ISSUE THREAD

**Issue by Soullivaneuh** [Soullivaneuh/php-cs-fixer-styleci-bridge#10]
Create a command to auto download missing fixer between PHP-CS-Fixer stable, on StyleCI/master fork.

Related to https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/issues/6#issuecomment-142606234.

**Comment by GrahamCampbell** [comment]
That won't work, because the fixer interface in 2.x might change from what it was in 1.x.

**Comment by GrahamCampbell** [comment]
And, actually, some stuff has changed internally in 2.x too, and more will change in the future too.

**Comment by GrahamCampbell** [comment]
The safest option is probably to actually just use StyleCI's php-cs-fixer fork.

**Comment by GrahamCampbell** [comment]
I'll be tagging v1.0.0 on there soon, so you shouldn't have to worry about me messing with the master branch anymore.

**Comment by GrahamCampbell** [comment]
I published it on packagist to make it easier to use.

**Comment by Soullivaneuh** [comment]
> That won't work, because the fixer interface in 2.x might change from what it was in 1.x.
> And, actually, some stuff has changed internally in 2.x too, and more will change in the future too.

Yeah I think I'll have difficulties too. This is just a issue for now, we will see that later. :+1: 

> I'll be tagging v1.0.0 on there soon, so you shouldn't have to worry about me messing with the master branch anymore.
> I published it on packagist to make it easier to use.

I'm not sure of what you are talking. You published your fixer for php-cs-fixer 1.x?

**Comment by Soullivaneuh** [comment]
I saw your repository: https://github.com/StyleCI/PHP-CS-Fixer

I'm asking the same questions: Why not just a repository with **only** your custom fixers with php-cs-fixer dependencies? This will permit to integrate it on this library with ease for example.

With this, I have to in any case parse fixers directories to manually get yours.

**Comment by Soullivaneuh** [comment]
And BTW, a same php support as official version for this one would be great. Otherwise, I can't link your library.

**Comment by Soullivaneuh** [comment]
ping @GrahamCampbell 

What do you think about https://github.com/Soullivaneuh/php-cs-fixer-styleci-bridge/issues/10#issuecomment-143201416?

**Comment by Soullivaneuh** [comment]
`StyleCI/PHP-CS-Fixer` repository does not exist anymore.

@GrahamCampbell Any chance to have your custom fixer? Tell me if absolutely not, I'll close the issue in this case.
